### DISCUSSION TITLE
RONI_defence - abstractdetect_poison not working

### DISCUSSION THREAD

**Discussion by RaMAT-44** [Trusted-AI/adversarial-robustness-toolbox#1279] [Q&A]
Hi Team,

Working on a use case to detect poison using RONI_defence on the adversarial data after RONIDefense is initialized called arbitrary method detect_poison. It returns an error on 

before_classifier.fit(x_suspect,y_suspect)

**Comment by beat-buesser** [comment]
Hi @TRKSasank Thank you very much for using ART! How can we reproduce the error you are observing?
